client/my-sites/upgrades/index.js

```diff
@@ -257,6 +257,11 @@ module.exports = function() {
 		);
 
 		page(
+			'/checkout/no-site/:receiptId',
```

Comment:
It should follow the existing pattern:
`/checkout/thank-you/no-site/:receiptId`

to make it work properly it needs to be moved before `/checkout/thank-you/:site/:receiptId?`.